src/chat/ContactSheet.tsx

```diff
@@ -62,7 +64,10 @@ const ContactSheet = forwardRef<View, Props>(({close}, focusRef) => {
         />
 
         {enable_intercom ? (
-          <>
+          <View
+            importantForAccessibility={'no-hide-descendants'}
+            accessibilityElementsHidden={true}
+          >
```

Comment:
Det er mulig å hente disse verdiene fra [`@atb/utils/accessibility`](https://github.com/AtB-AS/mittatb-app/blob/master/src/utils/accessibility.ts), er det bedre? Må også importeres, altså.

```suggestion
          <View {...screenReaderHidden}>
```

Ellers ok 👍

Reply:
awesome!